src/app/wallet/view.nim

```diff
@@ -15,6 +16,11 @@ QtObject:
       status: Status
       totalFiatBalance: string
       etherscanLink: string
+      safeLowGasPrice: string
+      standardGasPrice: string
+      fastGasPrice: string
+      fastestGasPrice: string
+      defaultGasLimit: string
```

Comment:
Why use `string` wouldn't it be easier as `int` or `float`?

Reply:
>  I think using int directly instead of string would save you a lot of parsing and headaches though.

*Actually* the values are already coming in as string so there's not that much of parsing going on. However, I did try using int or float, but parsing the values from String to either of those inside nim seems to mess up the value. For example float values represented as string and then parsed to float all turn into zeros.

Happy to revisit and change that again